### ISSUE TITLE
default_vocabularies Potential reuse of the MAL

### ISSUE THREAD

**Issue by athiasjerome** [MAECProject/schemas#117]
The MAL: A Malware Analysis Lexicon could potentially be interesting
http://repository.cmu.edu/cgi/viewcontent.cgi?article=1728&context=sei

**Comment by ikiril01** [comment]
@athiasjerome agreed! We've actually rolled some of these terms into our MalwareLabelVocab: http://maecproject.github.io/data-model/4.1/maecVocabs/MalwareLabelVocab-1.0/

However, we should probably take another pass at the MAL and make sure we align wherever appropriate.
